src/utilities/isBrowser.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/prefer-optional-chain
+export const isBrowser: boolean = typeof window !== 'undefined' && typeof window.document !== 'undefined'
```

Comment:
is this better than the `globalExists` utility used in prefect-design? I feel like that was even safer, as this presumes accessing `window` won't crash

Reply:
Idk, as long as we have tests that don't error in a browser environment or in a node environment I think this is sufficient. I think it's preferred to not use `eval` unless we absolutely have to.